### ISSUE TITLE
git detection doesn't seem to work for me

### ISSUE THREAD

**Issue by brentley** [jpignata/fargate#6]
I have a service running, created with this command:
`fargate service create web -v --lb fargate --port 3000`

```
$ fargate service list
NAME	IMAGE								CPU	MEMORY	LOAD BALANCER	DESIRED	RUNNING	PENDING
web	651426287273.dkr.ecr.us-east-1.amazonaws.com/web:20180102224503	256	512	fargate		1	1	0
```

Notice the image tag is date based, rather than being git-sha based.

yet I'm in a git repo:
```
$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```

**Comment by jpignata** [comment]
Thanks! I fixed this bug in master and will cut a new release tonight or tomorrow.

**Comment by jpignata** [comment]
Addressed in [v0.2.1](https://github.com/jpignata/fargate/releases/tag/v0.2.1).
